app/src/main/kotlin/com/generalmobile/app/gmnotes/db/converters/DateConverter.kt

```diff
@@ -11,7 +11,7 @@ class DateConverter {
     }
 
     @TypeConverter
-    fun dateToTimestamp(date: Date?): Long {
-        return date?.time ?: 0L
+    fun dateToTimestamp(date: Date?): Long? {
```

Comment:
Apparently both methods much have the same corresponding types (one was `Long`, the other was `Long?`), otherwise the compiler would give an error about not being able to find the typeconverter for the type.